extensions/integration-tests/src/objectExplorer.test.ts

```diff
@@ -27,3 +43,18 @@ if (context.RunTest) {
 		});
 	});
 }
+async function VerifyOeNode(server: TestServerProfile, timeout: number,expectedNodeLable: string[]) {
```

Comment:
typo `expectedNodeLable` should be `expectedNodeLabel`.  also missing ` ` after `,`.

Reply:
thanks